source/getting-started/quick-start.md

```diff
@@ -0,0 +1,262 @@
+
+We'll take the code created for us by the generator and add a `model()`
+method to the `Route`:
+
+```app/routes/scientists.js{+4,+5,+6}
+import Ember from 'ember';
+
+export default Ember.Route.extend({
+  model() {
+    return ['Marie Curie', 'Mae Jemison', 'Albert Hofmann'];
+  }
+});
+```
+
+(This code example uses the latest features in JavaScript, some of which
+you may not be familiar with. Learn more with this [overview of the
+newest JavaScript features][es6-bullet-points].)
+
+[es6-bullet-points]: https://ponyfoo.com/articles/es6
+
+In a route's `model()` method, you return whatever data you want to make
+available to the template. If you need to fetch data asynchronously, the
+`model()` method supports any library that uses [JavaScript
+Promises][promises].
+
+[promises]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise
+
+Now let's tell Ember how to turn that array of strings into HTML. Open
+the `scientists` template and add some Handlebars code to loop through the
+array and print it:
+
+```app/templates/scientists.hbs{+3,+4,+5,+6,+7}
+<h2>List of Scientists</h2>
+
+<ul>
+  {{#each model as |scientist|}}
+    <li>{{scientist}}</li>
+  {{/each}}
+<ul>
+```
+
+Here, we use the `each` helper to loop over each item in the array we
+provided from the `model()` hook and print it inside an `<li>` element.
+
+## Create a UI Component
+
+As your application grows and you notice you are sharing UI elements
+between multiple pages (or using them multiple times on the same page),
+Ember makes it easy to refactor your templates into reusable components.
+
+Let's create a `people-list` component that we can use
+in multiple places to show a list of people.
+
+As usual, there's a generator that makes this easy for us. Make a new
+component by typing:
+
+```sh
+ember generate component people-list
+```
+
+Copy and paste the `scientists` template into the `people-list`
+component's template and edit it to look as follows:
+
+```app/templates/components/people-list.hbs
+<h2>{{title}}</h2>
+
+<ul>
+  {{#each people as |person|}}
+    <li>{{person}}</li>
+  {{/each}}
+<ul>
+```
+
+Note that we've changed the title from a hard-coded string ("List of
+Scientists") to a dynamic property (`{{title}}`). We've also renamed
+`scientist` to the more-generic `person`, decreasing the coupling of our
+component to where it's used.
+
+Save this template and switch back to the `scientists` template. Replace all
+our old code with our new componentized version. Components look like
+HTML tags but instead of using angle brackets (`<tag>`) they use double
+curly braces (`{{component}}`). We're going to tell our component:
+
+1. What title to use, via the `title` attribute.
+2. What array of people to use, via the `people` attribute. We'll
+   provide this route's `model` as the list of people.
+
+```app/templates/people.hbs{-1,-2,-3,-4,-5,-6,-7,+8}
+<h2>List of Scientists</h2>
+
+<ul>
```

Comment:
@tomdale Re the wonky diff syntax, I'm not sure we're continuing to support that with the new redesign. It's not something we've had a need for up until now, but I've been seeing a few places here (and in the tutorial) where it might be handy.

@locks do you remember if we're adding in styling for diffs?  We'd been contemplating killing that I thought ...